packages/story-editor/src/components/errorBoundary/copyStoryDataToClipboard.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { __ } from '@googleforcreators/i18n';
+import {
+  Button,
+  BUTTON_SIZES,
+  BUTTON_TYPES,
+  BUTTON_VARIANTS,
+} from '@googleforcreators/design-system';
+import { useCallback } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { useStory } from '../../app/story';
+
+function CopyStoryDataToClipboard() {
+  const { reducerState } = useStory(({ internal }) => ({
+    reducerState: internal.reducerState,
+  }));
+
+  const { pages, current, selection, story } = reducerState;
```

Comment:
```suggestion
  const { pages, current, selection, story  } = useStory(({ internal }) => ({
    pages: internal.reducerState.pages,
    current: internal.reducerState.current,
    selection: internal.reducerState.selection,
    story: internal.reducerState.story,
  }));
```

Generally, we'd want to reduce the number of re-renders, which we can achieve by only returning what we need.

Example: If `reducer.foo` constantly changes, with the current version this component would always re-render. But with this suggestion it won't, because it's not actually used.